physikon/Infobrief_PhysiKon_2024.tex

```diff
@@ -0,0 +1,265 @@
+\documentclass[
+  paper=a4,
+  fontsize=12pt,
+  DIV=16,
+  headheight=30pt,
+  footheight=45pt,
+  headinclude,
+  parskip=half,
+]{scrartcl}
+
+\usepackage{fontspec}
+\setsansfont{Source Sans Pro}
+\renewcommand{\familydefault}{\sfdefault}
+\newfontfamily\geomfont{DejaVu Sans}
+\newcommand\checkbox{{\geomfont\symbol{"25A2}}}
+
+\usepackage{polyglossia}
+\setmainlanguage{german}
+
+\usepackage{microtype}
+\usepackage{graphicx}
+\usepackage[table]{xcolor}
+
+\usepackage{scrlayer-scrpage}
+\pagestyle{scrheadings}
+\setkomafont{pagehead}{\normalfont}
+\setkomafont{pagefoot}{\normalfont\footnotesize}
+\ohead{\includegraphics[height=1.5cm]{images/logo-pfeil-gruen.pdf}}
+\ihead{%
+  \large\textbf{Infobrief zur PhysiKon 2024}\\%
+}
+\cfoot{%
+  \parbox[c]{0.3\textwidth}{%
+    \input{pepadresse.tex}%
+  }%
+}
+
+\ofoot{%
+  \parbox[c]{0.3\textwidth}{%
+%   Bankverbindung\\
+  \input{pepbankverbindung.tex}%
+  }%
+}
+\ifoot{%
+  \includegraphics[height=1cm]{pep.pdf}\\
+  \url{www.pep-dortmund.org}
+}
+
+
+\usepackage{titling}
+\usepackage{booktabs}
+\usepackage{enumitem}
+\usepackage{csquotes}
+
+% \usepackage{xcolor}
+\usepackage{calc}
+\usepackage[colorlinks=true,urlcolor=blue!50!black]{hyperref}
+\renewcommand{\arraystretch}{1.5}
+
+\newcommand\MyTextField[2][]{\TextField[#1, backgroundcolor=black!10, charsize=0pt, borderwidth=0]{#2}}
+\renewcommand*{\LayoutTextField}[2]{\makebox[\widthof{#1: }][l]{#1: }%
+\raisebox{0.8\baselineskip}{\raisebox{-\height}{#2}}}
+
+
+\begin{document}
+
+\textbf{\Large Herzlich Willkommen zur PhysiKon-Woche 2024!}
+
+\vspace{1cm}
+Schön, dass Sie bei unserer Jobmesse PhysiKon mit an Board sind!
+In diesem Jahr wird die Jobmesse in einem hybriden Format vom 15. bis 18.04.2024 stattfinden.
+Im folgenden Schreiben können wir hoffentlich alle wichtigen Fragen zur Veranstaltung beantworten.
+Falls noch Fragen offen sind, melden Sie sich gerne bei uns!
+
+\section*{Kontakt}
+
+Sie erreichen das gesamte Team für organisatorische Rückfragen per Mail unter \href{mailto:physikon@pep-dortmund.org}{physikon@pep-dortmund.org}.
+In dringenden Fällen und während des Messetags erreichen Sie uns telefonisch unter
+\begin{itemize}
+  \item Till Willershausen: 0170 9858 082
+  \item Jan-Peter Herdieckerhoff: 0173 4532 166
```

Comment:
Jan Peter Wagner

Reply:
do you really want to have your private phone numbers in a public github repo?

Reply:
That is a real problem. Can we somehow remove this whole document from the git history?

Reply:
Can we make the repo private?

Reply:
If that helps, idk

Reply:
Lenas, Stefans and Thomas numbers by the way too for 2022 🙈